packages/teams-js/src/private/nestedAppAuth/nestedAppAuthBridge.ts

```diff
@@ -1,5 +1,8 @@
 import { v4 as generateUUID } from 'uuid';
 
+// Initial version of standalone nested app auth bridge
```

Comment:
Let's update the comment on how this version can be used by app developers?

Reply:
Updated. Please let me know if you have any further concern.